Userland/Libraries/LibWeb/HTML/ServiceWorker.idl

```diff
@@ -0,0 +1,29 @@
+#import <DOM/EventTarget.idl>
+#import <DOM/EventHandler.idl>
+#import <HTML/AbstractWorker.idl>
+#import <HTML/MessagePort.idl>
+
+// https://w3c.github.io/ServiceWorker/#service-worker-interface
```

Comment:
I think this URL is supposed to be: https://w3c.github.io/ServiceWorker/#serviceworker-interface